Unfettered news lines (#5935)
endless-sky/endless-sky: data/hai/hai news.txt

			"The Hai will pay you to rush cargo to human space and back. Don't die trying."
		word
			`"`

news "unfettered hai dialog"
	location
		government "Hai (Unfettered)"
	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Unfettered warlord"
			"Hai youth"
			"Hai warlord"
	message
		word
			`"`
		word
			"How did you get here? You should've been blasted to bits by our ships."
			"I came here because this is the only place in the galaxy where I feel wanted."
			"Weak human. Why do you dare to come to the land of the true Hai?"

> 			"Altered Hai are soft and weak-willed. They care not for true struggle or challenge."

Reviewer: The fact that there is a mythos story about the Hai being Altered probably shouldn't be used for naming. Hai are able to defect back and forth with relative ease and all Unfettered were part of the greater whole until recently (long after the supposed altering), so Fettered is probably better, implying an ideological limit rather than an actual physical limit.
```suggestion
			"Fettered Hai are soft and weak-willed. They care not for true struggle or challenge."
```

> 			"The last time a human called me 'cute', I ran him through with a machete."

Reviewer: They probably never met a human on an Unfettered world before for this to be true...

Author: Tried fixing it, is it better?

			"I seek challenge, and what greater challenge is there than to oppose the Drak, those with the power of gods?"
			"Sometimes I miss my family and friends back home, but then I remember that here I am a respected warrior. Back there I was a slave to the Drak."
			"Our ships can beat anything you stupid monkeys can make."
			"This sure beats being a factory worker."
			"Without change, a civilization will decay and fall. This will happen to the fettered Hai, as it has happened to countless empires throughout history."
			"What are you looking at, monkey?"
			"I can take on a human like you any day."
			"You looking for a fight, monkey?"
		word
			`"`
	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Hai youth"
	message
		word
			`"`
		word

> 			"At first I thought that joining the Unfettered would be fun. Now I can't go back or I'll be killed."

Reviewer: The various other texts which talk about how the Hai view the Unfettered suggest that this is in face entirely false.

Author: I meant to imply that other Unfettered would try to kill him for 'deserting'.

Reviewer: Ah yeah that was not at all clear. I'm really not inclined to think that *death* is the likely outcome though, even on the Unfettered side of things.

Author: I'll try fixing this tomorrow then.

Author: I edited it, is it okay now?

			"I feel like the only Hai on this planet who regrets joining the Unfettered."
			"I want to go home."
		word
			`"`
news "unfettered hai after wanderers invasion"
	location
		government "Hai (Unfettered)"
	to show
		has "Wanderers Invaded 0: done"
	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Unfettered warlord"
			"Hai youth"
			"Hai warlord"
	message
		word
			`"`
		word
			"We shall conquer the worlds to the north as is our right. It is inevitable that the strong shall defeat the weak."
			"I have fought with the stupid birds to the north. They have no knowledge of proper battle. Our war with them will be swift."
			"Those overgrown birds have fantastic weaponry, yet they are too cowardly to use it."
			"I wonder why the birds do not simply fight back. Their weapons are clearly capable of doing so."

> 			"I wish I could get my hands on some of that Wanderer tech."
> 			"Wanderer worlds are pristine! We have won a great prize in our battle against the stupid birds."

Reviewer: It's unclear if the Unfettered a) know their name, and b) would consider it appropriate to use as such.
```suggestion
			"I wish I could get my hands on some of that bird tech."
			"The bird worlds are pristine! We have won a great prize in our battle against the stupid birds."
```

			"The true Hai will no longer be confined in their Drak-imposed cage."
			"I'll see great glory in fighting the weak-willed birds."

> 			"The worlds up north are nothing like the ones we were confined to. On the Wanderer worlds, the air is clean and the water is as clear as glass."

Reviewer: Same deal.
```suggestion
			"The worlds up north are nothing like the ones we were confined to. On the bird worlds, the air is clean and the water is as clear as glass."
```

		word
			`"`
news "unfettered hai after wanderers invasion regret"
	location
		government "Hai (Unfettered)"
	to show
		has "Wanderers Invaded 0: done"
	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Hai youth"
	message
		word
			`"`
		word

> 			"I had to shoot down a Wanderer ship full of civilians when my captain decided to invade their space. I don't know if I can take this anymore."
> 			"The Wanderers hardly try to fight back. It makes me wonder if they want us to conquer their planets."

Reviewer: Birds, but also, I'm not sure that first one is entirely true. We haven't heard of any such atrocities being performed against the Wanderers, they seemed to be pretty effective at evacuation.
```suggestion
			"I had to shoot down a bird ship full of civilians when my captain decided to invade their space. I don't know if I can take this anymore."
			"The birds hardly try to fight back. It makes me wonder if they want us to conquer their planets."
```

Author: Correct me if I'm wrong, but the player doesn't take part in every evacuation. After all, we're talking planets full of people. I think it's possible that there were some evacuations that didn't go well and were attacked.

Reviewer: True, but there aren't a lot of wanderers, I feel like that would get a lot of attention.

Author: Are there any solid population figures for the Wanderers?

Reviewer: In `mission "Unfettered Jump Drive 3"` the Unfettered say, "They are an old and strong species, but few in number, and those worlds are ours by right."

There are no hard numbers, but this implies that the Wanderers are fewer in number than even the Unfettered.

Author: I think that it's plausible that between the Unfettered attacks, Iktat Rek's Molt, and the general chaos of trying to rebuild an army from the ground up in a short timeframe, that something like this happening could just not be mentioned to the player. It doesn't necessarily have to be a large ship, either. I could change it to be some small merchant ship, if that would work.

Reviewer: Mmm, at present the wording implies it's an evacuation ship - which is one of the limited number of deep river transports with 166 people on board (or after any one of the rebalances happen, somewhere between 800 and 1400 people, outcome tbd).  

So yeah maybe "one of their merchant ships that was mistaken for a combatant" or something is still like a "oh dear we blew up a bunch of innocent civilians" without being a "oh we actively committed deliberate war crimes against the biggest transport in the galaxy worth of civilians".

Author: I edited the line, does it work better now?

Reviewer: Oh yes, I like the new version.

			"I don't think it's right that we attack those that have surrendered. What honor is there in killing civilians?"
			"I came here to do battle with warriors, not with merchants and cowardly birds."
			"What challenge is there in killing the weak? I desire to battle with those who are worthy."

> 			"What we're doing up north isn't right, but if I try to speak against it, others call me 'coward' and threaten to kill me."

Reviewer: Again, actual narrative so far suggests that killing each other isn't done often or personally.
```suggestion
			"What we're doing up north isn't right, but if I try to speak against it, others call me 'coward' and tell me to ship myself home"
```

			"I was held prisoner on a Wanderer world. They fed me and took care of me. The only thing they asked me to do in return was to teach them the Hai langauge. If it were the other way around, I'd be dead."
			"I was held prisoner on a Wanderer world. They said that we were welcome to take their planets, if it was their fate. It must be a trap."

> 			"I was held prisoner on a Wanderer world. Even though I attacked them, they were so kind to me. I don't want to fight them anymore, I'd feel too guilty."

Reviewer: The Wanderers do not appear to have taken prisoners in the narrative. Also, birds.

Author: I was basing it off of this quote from Iktak Rek: `Recently we managed to [take, capture] some of them as prisoners," he says.`, when asked by the player how the Wanderers learned the Hai language.

Reviewer: Huh, I must've totally missed that in my reading. Damn

Author: You don't mind if I resolve this, right?

Reviewer: So it's worth noting that they never identify themselves as the Wanderers at the start, the player can see the naming of them in the game, and so the game simply assumes that the player knows they're "Wanderers" and treats you accordingly. Though amusingly, the mission where they refer to having captured prisoners is the first time they refer to themselves as Wanderers and it's not just meta knowledge.
I'm going to say yes you can resolve this, but I think it would be interesting to include a news item or two about the use of the name. It seems to me that the Unfettered must have been raiding them for a while, and have established a habit of referring to them in some manner before they ever knew they were "Wanderers".
Perhaps there could be some entries implying a split, those who are still bent on invading merely referring to them as birds, and those who have developed some sympathy using the less popular "Wanderers"?

Author: I'm going to assume that the Wanderers call themselves that, or whatever is the equivalent in their local language. I added a couple of news lines that referred to this.

Reviewer: I like them.

	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Unfettered warlord"
			"Hai youth"
			"Hai warlord"
	message
		word
			"A Hai looks at you and shouts something in their language. You get the impression that it isn't pleasant."
			"You pass by a Hai with a false tail. You try not to stare as you walk past."
			"At a booth, you see a Hai selling stolen ship weaponry for outrageous prices."
			"At a booth, you see a Hai selling various types of unpleasant-smelling substances. Judging by the effects it has on those who consume them, you guess that they're drugs."

news "unfettered hai no dialog group"
	location
		government "Hai (Unfettered)"
	name
		word
			"Group of Unfettered"
			"Group of Hai"
			"Group of youth"
	message
		word
			"You see a group of Hai confront another. They shout at each other and bring out weapons, before backing off."
			"You see a group of Hai confront another. They shout and bring out weapons, before lunging at each other. You walk away before they notice you."
			"You hear yelling in the Hai language. When you turn around, you see that two Hai are facing off against each other, as onlookers egg them on. You turn away before it gets ugly."

> 			"A gang of Hai go around, visibly carrying weapons. Older Hai 'donate' money to them."

Reviewer: This last line feels very incongruous. The Hai are, in general, quite fit very late into their lives (which are long), and the evidence of the Hai's comments about the Unfettered seems to suggest that they're welcome to come home and return to the fold any time. As such I doubt very much that there's a class of older Hai on Unfettered worlds being extorted for protection money.

			"The aliens to the north call themselves the Wanderers. Overgrown birds is more like it."
		word
			`"`
news "unfettered hai after wanderers invasion regret"
	location
		government "Hai (Unfettered)"
	to show
		has "Wanderers Invaded 0: done"
	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Hai youth"
	message
		word
			`"`
		word
			"I had to shoot down a surrendering Wanderer ship when my captain decided to invade their space. It was small, and didn't pose a threat, but my captain made me do it anyways. I don't know if I can take this anymore."
			"The birds hardly try to fight back. It makes me wonder if they want us to conquer their planets."
			"I don't think it's right that we attack those that have surrendered. What honor is there in killing civilians?"
			"I came here to do battle with warriors, not with merchants and cowardly birds."
			"What challenge is there in killing the weak? I desire to battle with those who are worthy."
			"What we're doing up north isn't right, but if I try to speak against it, others call me 'coward' and tell me to ship myself home."

> 			"I was held prisoner on a Wanderer world. They fed me and took care of me. The only thing they asked me to do in return was to teach them the Hai langauge. If it were the other way around, I'd be dead."

Reviewer 2: Just a typo:
```suggestion
			"I was held prisoner on a Wanderer world. They fed me and took care of me. The only thing they asked me to do in return was to teach them the Hai language. If it were the other way around, I'd be dead."
```

			"The Hai will pay you to rush cargo to human space and back. Don't die trying."
		word
			`"`

news "unfettered hai dialog"
	location
		government "Hai (Unfettered)"
	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Unfettered warlord"
			"Hai youth"
			"Hai warlord"
	message
		word
			`"`
		word
			"How did you get here? You should've been blasted to bits by our ships."

> 			"I came here because this is the only place in the galaxy where I feel wanted."

Author: Too emo?

			"Weak human. Why do you dare to come to the land of the true Hai?"
			"Fettered Hai are soft and weak-willed. They care not for true struggle or challenge."
			"You call me 'cute', and I'll run you through with a machete."
			"I seek challenge, and what greater challenge is there than to oppose the Drak, those with the power of gods?"
			"Sometimes I miss my family and friends back home, but then I remember that here I am a respected warrior. Back there I was a slave to the Drak."
			"Our ships can beat anything you stupid monkeys can make."
			"This sure beats being a factory worker."
			"Without change, a civilization will decay and fall. This will happen to the fettered Hai, as it has happened to countless empires throughout history."
			"What are you looking at, monkey?"
			"I can take on a human like you any day."
			"You looking for a fight, monkey?"
			"Last week we plundered a human ship that was stupid enough to come up here. Who knows, maybe you'll be next."
			"We just plundered a merchant convoy that was loaded with food. We'll feast well tonight."
		word
			`"`
news "Unfettered Regret"

> 	name

Reviewer 3: You're adding six news types to Unfettered world, two of which are Hai showing regret, so a third of the time the news you get would be about Unfettered regretting their actions, which seems like a dangerously low morale. We could put random conditions in the to shows of these news types to reduce the odds of seeing an Unfettered expressing regret.
```suggestion
	to show
		random < 10
	name
```

			"The birds hardly try to fight back. It makes me wonder if they want us to conquer their planets."
			"I don't think it's right that we attack those that have surrendered. What honor is there in killing civilians?"
			"I came here to do battle with warriors, not with merchants and cowardly birds."
			"What challenge is there in killing the weak? I desire to battle with those who are worthy."
			"What we're doing up north isn't right, but if I try to speak against it, others call me 'coward' and tell me to ship myself home."
			"I was held prisoner on a Wanderer world. They fed me and took care of me. The only thing they asked me to do in return was to teach them the Hai language. If it were the other way around, I'd be dead."
			"I was held prisoner on a Wanderer world. They said that we were welcome to take their planets, if it was their fate. It must be a trap."
			"I was held prisoner on a Wanderer world. Even though I attacked them, they were so kind to me. I don't want to fight them anymore, I'd feel too guilty."

> 			"I was held prisoner by the aliens up north. They call themselves the Wanderers, but everyone here just calls them 'birds'."

Reviewer 3: ```suggestion
			"I was held prisoner by the aliens up north. They call themselves the Wanderers, but everyone here just calls them 'birds.'"
```

			"Unfettered warlord"
			"Hai youth"
			"Hai warlord"
	message
		word
			`"`
		word
			"How did you get here? You should've been blasted to bits by our ships."
			"I came here because this is the only place in the galaxy where I feel wanted."
			"Weak human. Why do you dare to come to the land of the true Hai?"
			"Altered Hai are soft and weak-willed. They care not for true struggle or challenge."
			"The last time a human called me 'cute', I ran him through with a machete."
			"I seek challenge, and what greater challenge is there than to oppose the Drak, those with the power of gods?"
			"Sometimes I miss my family and friends back home, but then I remember that here I am a respected warrior. Back there I was a slave to the Drak."
			"Our ships can beat anything you stupid monkeys can make."
			"This sure beats being a factory worker."
			"Without change, a civilization will decay and fall. This will happen to the fettered Hai, as it has happened to countless empires throughout history."
			"What are you looking at, monkey?"
			"I can take on a human like you any day."
		word
			`"`
	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Hai youth"

> 		word

Reviewer 4: I'm fairly sure that you need to include `message` where these are and indent what is below it.

			`"`
		word
			"At first I thought that joining the Unfettered would be fun. Now I can't go back or I'll be killed."
			"I feel like the only Hai on this planet who regrets joining the Unfettered."
			"I want to go home."
		word
			`"`
news "unfettered hai after wanderers invasion"
	location
		government "Hai (Unfettered)"
	to show
		has "Wanderers Invaded 0: done"
	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Unfettered warlord"
			"Hai youth"
			"Hai warlord"

> 		word

Reviewer 4: As with before, I'm fairly sure that you need to include `message` where these are and indent what is below it.

			`"`
		word
			"We shall conquer the worlds to the north as is our right. It is inevitable that the strong shall defeat the weak."
			"I have fought with the stupid birds to the north. They have no knowledge of proper battle. Our war with them will be swift."

news "unfettered hai no dialog"
	location
		government "Hai (Unfettered)"
	name
		word
			"Unfettered fighter"
			"Unfettered youth"
			"Unfettered warlord"
			"Hai youth"
			"Hai warlord"

> 		word

Reviewer 4: As with before, I'm fairly sure that you need to include `message` where these are and indent what is below it.
